tagsets/expregister.php

```diff
@@ -335,11 +335,23 @@ function expregister__cancel($participant,$session) {
     $done=experimentmail__experiment_cancellation_mail($participant,$session);
 }
 
-
-
+function expregister__language_test_signed_select_field($postvarname,$selected,$hidden=array(),$show_color=true) {
+    $language_test=[ 'y', 'n', 'o' ];
```

Comment:
What does the value 'o' stand for?

Reply:
o is the initial value. as a test can be not taken, passed, failed.